.travis/install_bower.sh

```diff
@@ -1,5 +1,3 @@
-#!/bin/bash
-set -e
-set -x
+#!/bin/bash -ex
```

Comment:
The shebang line was already in the original code, I just amended it. And I actually do like that with `bash install_bower.sh` the options do not take effect as that gives you an easy way to disable them without modifying the script. Although that might be a rather academic use case.